src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -248,6 +248,8 @@ void IgnRenderer::BroadcastRightClick()
 
   events::RightClickToScene rightClickToSceneEvent(pos);
   App()->sendEvent(App()->findChild<MainWindow *>(), &rightClickToSceneEvent);
+  events::RightClickOnScene rightClickOnSceneEvent(this->dataPtr->mouseEvent);
+  App()->sendEvent(App()->findChild<MainWindow *>(), &rightClickOnSceneEvent);
```

Comment:
Should we do the same for the left click?

Reply:
it's done here

https://github.com/ignitionrobotics/ign-gui/blob/bde1977b4600602fb63f08d74cfbeeaf5bd3919b/src/plugins/minimal_scene/MinimalScene.cc#L168-L169

Reply:
Ah thanks. But it's strange that these are not done in the same place. The right click is in `Broadcast` while the left click is in `HandleMouseViewControl`. Do you know why they're not handled the same way?